extensions/amp-ad/0.1/amp-ad-xorigin-iframe-handler.js

```diff
@@ -162,24 +157,32 @@ export class AmpAdXOriginIframeHandler {
           .then(() => this.noContent_());
     }
 
-    // Set iframe initially hidden which will be removed on load event +
-    // post message.
-    setStyle(this.iframe, 'visibility', 'hidden');
-
-    this.element_.appendChild(this.iframe);
-    return timerFor(this.baseInstance_.win).timeoutPromise(TIMEOUT_VALUE,
-        this.adResponsePromise_,
-        'timeout waiting for ad response').catch(e => {
-          this.noContent_();
-          user().warn('AMP-AD', e);
-        }).then(() => {
-          if (this.iframe) {
-            setStyle(this.iframe, 'visibility', '');
-            if (this.baseInstance_.emitLifecycleEvent) {
-              this.baseInstance_.emitLifecycleEvent('adSlotUnhidden');
+    if (opt_isA4A) {
```

Comment:
any reason to move this code? A4A creative in friendly iframe will not send 'render-start' event, so there's no need to listen to it.